io/pci/EEH.py

```diff
@@ -134,6 +143,21 @@ def test_eeh_basic_pe(self):
                                 break
                             else:
                                 self.log.info("PE recovered successfully")
+                    time.sleep(10)
+                    if self.pci_class_name == 'scsi_host':
+                        paths_after_eeh = self.get_paths(self.pci_device)
+                        host_state_after_eeh = self.get_host_state(
+                                self.pci_device)
+                        self.log.info("Paths after EEH: %s",
+                                      paths_after_eeh)
+                        self.log.info("hosts state after EEH: %s",
+                                      host_state_after_eeh)
+                        if paths_before_eeh != paths_after_eeh:
```

Comment:
I don't think this verification will yield the right validation.
always check the count and status of each file.
number of paths before EEH for that pci_address should be eaqual to number of paths after eeh trigger.
and the stats of each paths before and after should be active.
only in one case paths will not be in active state i,e on nth time eeh trigger. so you have to include that corner case too here

Reply:
@Naresh-ibm Array comparison is doing same thing as you said. I will include corner case as suggested.